spring-cloud-dataflow-server-core/src/main/java/org/springframework/cloud/dataflow/server/controller/AboutController.java

```diff
@@ -15,28 +15,20 @@
  */
 package org.springframework.cloud.dataflow.server.controller;
 
-import java.util.ArrayList;
```

Comment:
Please move the j`ava.util.*` back up to here. The formatter will bark about this. The [code style instructions](https://github.com/spring-cloud/spring-cloud-dataflow/wiki/Working-with-the-Code#code-formatting) are in the Wiki as well - you may want to set those up.

Reply:
Thanks... done...